doc/api/configuration.md

```diff
@@ -240,4 +240,14 @@ print(res)
 %config SqlMagic.style = "SINGLE_BORDER"
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
-```
\ No newline at end of file
+```
+
+## Loading configuration settings
+
+You can define configurations in a `pyproject.toml` file and automatically load the configurations when you run `%load_ext sql`. It will try to find the file recursively from the current to root directory. If the file is not found, default values will be used. A sample `pyproject.toml` could look like this:
```

Comment:
I'm not sure this is relevant for our users: `It will try to find the file recursively from the current to root directory`. They just need to know that they can set their configuration via `pyproject.toml`.

Reply:
Removed `It will try to find the file recursively from the current to root directory`